### PR TITLE
Docs: Add link to awesome-eslint in integrations page

### PR DESCRIPTION
**Pull request by vitorbal**
**What issue does this pull request address?**
Add a link to awesome-eslint in the integrations page. I think they do a good job at curating a bunch of integrations for ESLint, and they seem to have good traction, so I believe it would be helpful to suggest people to take a look in there as well.

**What changes did you make? (Give an overview)**
I added a new section to the end of the integrations page that suggests awesome-eslint as another source for useful ESLint integrations.

**Is there anything you'd like reviewers to focus on?**
Wording suggestions, I guess.

### CODE REVIEW

**Comment by platinumazure** [comment]
LGTM, but I want to get another set of eyes on this.

File: docs/user-guide/integrations.md [hunk 1]
```diff
@@ -51,3 +51,7 @@
 * [React](https://github.com/yannickcr/eslint-plugin-react)
 
 … and many more published on npm with the [eslintplugin](https://www.npmjs.com/browse/keyword/eslintplugin) keyword.
+
+## Awesome ESLint
+
+You can find a curated list of other popular integrations for ESLint in the [awesome-eslint](https://github.com/dustinspecker/awesome-eslint) github repository.
```
Comment (nzakas) [inline docs/user-guide/integrations.md]: GitHub (capitalization)

File: docs/user-guide/integrations.md [hunk 2]
```diff
@@ -51,3 +51,7 @@
 * [React](https://github.com/yannickcr/eslint-plugin-react)
 
 … and many more published on npm with the [eslintplugin](https://www.npmjs.com/browse/keyword/eslintplugin) keyword.
+
+## Awesome ESLint
```
Comment (nzakas) [inline docs/user-guide/integrations.md]: I'd maybe have this header be a bit more generic, as it's not obvious what this means. Maybe "Other Integration Lists"?

Comment (vitorbal) [inline docs/user-guide/integrations.md]: okay!

**Comment by vitorbal** [comment]
@nzakas thanks for the feedback! Updated with your suggestions.

**Comment by platinumazure** [comment]
LGTM again.

**Comment by kaicataldo** [comment]
LGTM

**Pull request merged**